doc/operator_auth.md

```diff
@@ -0,0 +1,163 @@
+ Authentication protocol for VM owner
+=======================================
+This custom protocol allow a user (owner of a VM) to securely authenticate to a CRN, using their Ethereum wallet.
+This scheme was designed in a way that's convenient to be integrated in the console web page.
+
+It allows the user to control their VM. e.g : stop reboot, view their log, etc…
+
+## Overview
```

Comment:
```suggestion
## Motivations
This protocol ensures secure authentication between a blockchain wallet owner and an aleph.im compute node.

Private key access is typically gated by prompts requiring manual approval for each signing operation. With hardware wallets, users are prompted both by the software on their device and the hardware wallet itself.

## Overview
```

Reply:
not sur I understand the second paragraph. is that a llm suggestion?

Reply:
What I mean there is that when using a client (CLI or web) with a blockchain wallet, the user usually has to sign every operation with a prompt.

For example with Metamask:

![Screenshot 2024-09-17 at 10-42-37 MetaMask](https://github.com/user-attachments/assets/6d5a0f25-297a-45f9-856b-7b8874d661fb)

When the private key is stored in a hardware wallet, it is one prompt on screen plus one on the hardware wallet.

![image](https://github.com/user-attachments/assets/8d08d08d-7674-4cad-bdac-e01aa5b01015)

This is very annoying to do for every operation on a CRN (access logs, ...).

I want to explain why this intermediate temporary key pair is created.